### PR TITLE
Wrapped acf_set_options_page_menu in if function_exists statement to …

### PR DESCRIPTION
**Pull request by zackkrida**
…prevent error on initilization and update. 

Pretty straightforward, see http://www.advancedcustomfields.com/resources/acf_set_options_page_menu/ for example.

### CODE REVIEW

**Comment by atticoos** [comment]
oh yes thanks, forgot that one

File: lib/init.php [hunk 1]
```diff
@@ -40,10 +42,10 @@ function admin_menu_overrides() {
   acf_add_options_sub_page(array(
     'title' => 'Event Page Options',
     'parent' => 'edit.php?post_type=event'
-  ));
+    ));
```
Comment (atticoos) [inline lib/init.php]: Can we avoid the editor diffs? Will spread like wildfire

Comment (zackkrida) [inline lib/init.php]: sure thing

File: lib/init.php [hunk 2]
```diff
@@ -31,7 +31,9 @@ function nav_menus_init() {
 add_action('init', __NAMESPACE__ . '\\nav_menus_init');
 
 function admin_menu_overrides() {
-  acf_set_options_page_menu('Theme Options');
+  if(function_exists('acf_set_options_page_menu')) {
```
Comment (atticoos) [inline lib/init.php]: Looks like the build failed b/c it enforces a space here
`if (condition)`
vs
`if(condition)`

**Comment by zackkrida** [comment]
d'oh! you think i'd read the ruleset before submitting a pr.

**Comment by atticoos** [comment]
Ha, simply b/c 3rd commit's a charm is all

**Comment by atticoos** [comment]
:+1:

**Pull request merged**